### ISSUE TITLE
How to configure the size of arrow header?

### ISSUE THREAD

**Issue by tongdu** [stefankueng/demohelper#9]
How to modify the arrow header, because of the default header is so tiny like a straight line. Thanks.

**Comment by dcasonic** [comment]
Yes! I would like do this too. Or draw arrows like in 1.3 version.

**Comment by iberck** [comment]
I would like do this too...

+1
